lib/jsdom/living/helpers/focusing.js

```diff
@@ -60,6 +56,16 @@ exports.isFocusableAreaElement = elImpl => {
     // attribute is not implemented.
   }
 
+  if (elImpl._namespaceURI === HTML_NS || elImpl._namespaceURI === SVG_NS) {
```

Comment:
`_namespaceURI` and `_localName` should probably be destructured:
```js
const { _namespaceURI, _localName } = elImpl;
```

Reply:
No, underscore-prefixed local variables are bad style. (Please remember to avoid unsolicited drive-by reviews on other people's projects.)